src/server/config.js

```diff
@@ -16,6 +16,15 @@ const SERVER_SUCCESS = {
     UPDATE_OK: { MESSAGE: 'Update Successful', CODE: 201 }
 }
 
+const PRISMA_ERROR = {
```

Comment:
In this global store you use POJOs - which contradicts the convention i saw in the client side store.

Prefer this approach, but remember to be consistent.